src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -0,0 +1,50 @@
+@echo off 
+setlocal
+
+REM 
+REM Start WCF Self Hosted Service on local machine
+REM
+
+REM Build tools
+call BuildWCFSelfHostedService.cmd
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while building WCF Self hosted Service.
+    set __EXITCODE=%ERRORLEVEL% 
+    goto :done
+  )
+call BuildCertUtil.cmd
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while building the Certificate generator.
+    set __EXITCODE=%ERRORLEVEL%
+    goto :done
+  )
+
+
+REM Config Certs
+REM we need the direcotry to save the test.crl file. We are investigate a way to get rid of it
+REM
+md c:\wcftest
+pushd ..\..\..\..\bin\Wcf\tools\CertificateGenerator
+call CertificateGenerator.exe
+set __EXITCODE=%ERRORLEVEL%
+popd
+if NOT [%__EXITCODE%]==[0] (
+    echo Warning: An error occurred while running certificate generator.
+  )
+
+powershell -NoProfile -ExecutionPolicy unrestricted .\ConfigHttpsPort.ps1
+if NOT [%ERRORLEVEL%]==[0] (
+    echo Warning: An error occurred while configuration https port.
+  )
+
+REM Start the self hosted WCF Test Service
+pushd ..\..\..\..\bin\Wcf\tools\SelfHostedWcfService
+call SelfHostedWcfService.exe
```

Comment:
Not blocking for now, but it should pass %* to SelfHostedWcfService.exe, which will probably parse TestProperties.  This would allow us to wire StartWcfService.cmd directly into build.cmd and flow msbuild properties all the way down.  This is how the bridge plumbing works today.